frontend/src/Judge/containers/VotePanel.js

```diff
@@ -0,0 +1,38 @@
+import { graphql, compose } from 'react-apollo'
+import { connect } from 'react-redux'
+
+import VotePanel from '../components/VotePanel'
+import SendVote from '../mutations/sendVote.graphql'
+import GetVote from '../queries/entryVote.graphql'
+
+const withMutations = compose(
+  graphql(SendVote, {
+    props: ({ mutate, ownProps }) => ({
+      vote: (value) => mutate({
+        variables: {
```

Comment:
Hmm, I think the `refetchQueries` thing might have some use here... see [`frontend/src/Admin/containers/AssignJudgesTable.js`](https://github.com/robmcl4/A-B-S-T-R-A-C-T-I-O-N/blob/master/frontend/src/Admin/containers/AssignJudgesTable.js#L58) for how we used it previously. I'd like to hear Bill's opinion, since he put that there in the first place... i'm not too sure what it does exactly.

Reply:
Apollo's mutations send and forget. You can specify `refetchQueries` to get data back that may have changed because of your mutation. It's not really the direction Apollo steers you in – they'd rather have you update the local cache rather than make another network request – but because it was easier to implement / reason about and our app is small, I used `refetchQueries` anyways.

We'll need it here to display what choice a Judge made, and again, it's probably easier to use `refetchQueries` than to do cache updates, though I would think this would be more of a candidate for cache updates than the Admin side (since votes are scoped to a particular Judge).